ContactDetailsApi/V2/UseCase/FetchAllContactDetailsByUprnUseCase.cs

```diff
@@ -29,7 +29,10 @@ public FetchAllContactDetailsByUprnUseCase(ITenureDbGateway tenureGateway, IPers
         private async Task<IEnumerable<TenureInformation>> GetTenures()
         {
             var tenures = await _tenureGateway.GetAllTenures().ConfigureAwait(false);
-            tenures = tenures.Where(x => x.TenuredAsset?.Uprn != null).ToList();
+            tenures = tenures.Where(x => x.TenuredAsset?.Uprn != null && x.IsActive == true)
+                            .GroupBy(x => x.TenuredAsset.Uprn)
+                            .Select(x => x.First())
```

Comment:
`.First()` will throw an exception if nothing to return. 
`.FirstOrDefault()` will not

Reply:
https://github.com/LBHackney-IT/contact-details-api/pull/131